hrjobcontract/templates/CRM/Hrjobcontract/Import/Form/MapField.tpl

```diff
@@ -28,13 +28,13 @@
 <div class="crm-block crm-form-block crm-import-mapfield-form-block">
 {* WizardHeader.tpl provides visual display of steps thru the wizard as well as title for current step *}
 {include file="CRM/common/WizardHeader.tpl"}
-<div id="help">
-    <p>{ts}Review the values shown below from the first 2 rows of your import file and select the matching CiviCRM database fields from the drop-down lists in the right-hand column. Select '- do not import -' for any columns in the import file that you want ignored.{/ts}</p>
+  <div class="help">
+    <p>{ts}Review the values shown below from the first 2 rows of your import file and select the matching CiviHR database fields from the drop-down lists in the right-hand column. Select '- do not import -' for any columns in the import file that you want ignored.{/ts}</p>
     {if $savedMapping}
     <p>{ts}Click 'Load Saved Field Mapping' if data has been previously imported from the same source. You can then select the saved import mapping setup and load it automatically.{/ts}<p>
-    {/if}
+      {/if}
```

Comment:
Please remove the spaces to make indentation of the closing {if} match the indentation of the opening {if}